mmv1/products/integrations/Client.yaml

```diff
@@ -113,3 +120,4 @@ properties:
       User input run-as service account, if empty, will bring up a new default service account.
     immutable: true
     ignore_read: true
+    deprecation_message: '`run_as_service_account` is deprecated and will be removed in a future major release.'
```

Comment:
Is there an alternative users should use instead?

Reply:
This field is a no-op and is not used anymore by the service. There are no alternative to this field.